irc.html

```diff
@@ -13,21 +13,21 @@ <h1>Channels</h1>
   </p>
   <ul>
     <li>
-      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#Minetest</a></strong>
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel where everyone is welcome to discuss the game or request support.
     </li>
+    
     <li>
-      <strong>
-        <a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a>
-      </strong>
-      (<a href="http://irc.minetest.net/minetest-dev">logs</a>):
-      development channel, only serious stuff; note that webchat is muted due to too much spam/flood.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-dev">#Minetest-Dev</a></strong>
+        (<a href="http://irc.minetest.net/minetest-dev">Logs</a>):
+        Development channel, strictly engine or minetest_game development talk only; note that webchat is muted due to too much spam/flood.
     </li>
+    
     <li>
-      <strong>#minetest-hub</strong>
-      (<a href="http://irc.minetest.net/minetest-hub">logs</a>):
-      A moderated channel for people who regularly improve Minetest by developing, modding, hosting servers and otherwise.
-      See <a href="http://hub.minetest.net/info">the channel's website</a> for more information.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-hub">#Minetest-Hub</a></strong>
+        (<a href="http://irc.minetest.net/minetest-hub">Logs</a>):
+        A moderated channel for people who regularly improve Minetest by developing, modding, hosting servers and otherwise.
```

Comment:
As above, if possible split long line in code.

Reply:
As above.